docs/src/intro-js.md

```diff
@@ -54,13 +56,18 @@ pnpm create playwright
 
 </Tabs>
 
+(This command either initializes a new project with Playwright, or adds Playwright setup to your current project.)
```

Comment:
As a rule of thumb, we try to avoid parenthesis. I think we can just amend the first "npm, yarn or pnpm" sentence, and mention this.

Reply:
Sure, that's understandable.

Reply:
Done but I couldn't do it exactly as you asked. Let me know if it looks better.